tests/unit/test_charm.py

```diff
@@ -165,3 +242,47 @@ def test_ingress_relation_created(
         "port": port,
         "strip-prefix": "true",
     }
+
+
+def test_config_updated_on_ingress_relation_joined(
+    harness, mocked_kubernetes_service_patcher
+) -> None:
+    harness.begin()
```

Comment:
I recommend to use `setUp` in order to avoid calling `begin()` and having to mock `kubernetes_service_patcher` in every unit test